lesson08/activity/starter/activity08/src/App.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div id="app">
+    <router-view/>
+  </div>
+</template>
+
+<script>
+
+</script>
+
+<style lang="scss">
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+}
+
+#nav {
+  padding: 30px;
```

Comment:
For maximum responsivity, we should always use relative units (rem, em, %) for layout and sizing.